app/src/main/java/org/gdg/frisbee/android/fragment/NewsFragment.java

```diff
@@ -64,10 +69,10 @@
 
     private NewsAdapter mAdapter;
 
-    public static NewsFragment newInstance(String plusId) {
+    public static NewsFragment newInstance(final String plusId) {

```

Comment:
In C++ yes, not in Java. In Java you can always change the object even if it's final, in C++ you can't, you just can't change the reference, so it's pointless since the reference can only be changed inside the method, nobody else can change it. If the method is big (which shouldn't be) it might be helpful but for small methods it's pointless.